scripts/dracula.sh

```diff
@@ -180,7 +180,7 @@ main()
 
     elif [ $plugin = "weather" ]; then
       IFS=' ' read -r -a colors <<< $(get_tmux_option "@dracula-weather-colors" "orange dark_gray")
-      script="#($current_dir/weather_wrapper.sh $show_fahrenheit $show_location $fixed_location)"
+      script="#($current_dir/weather_wrapper.sh $show_fahrenheit $show_location "\"$fixed_location\"")"
```

Comment:
Couldn't this just be `"${fixed_location}"`? https://stackoverflow.com/questions/8748831/when-do-we-need-curly-braces-around-shell-variables

Reply:
I tried `"${fixed_location}"` but no luck, still need escape quote when have space in like this in `.tmux.conf`
`set -g @dracula-fixed-location "New York"`

Just `New` got inside and the weather also wrong
![image](https://user-images.githubusercontent.com/71129995/235068867-0f3a0ec0-23b3-430b-b884-ccf29bcde7ed.png)